client/src/pages/orders.vue

```diff
@@ -81,20 +81,21 @@ const updateItemEvent = async (id: string, eventId: string) => {
   });
 };
 
-const eventOrders = ref([
-  {
-    status: false,
-    name: "スズキ",
-    timeofreceipt: "2022/09/09 14:00 - 15:00",
-    recievedtime: "14:34",
-    address: "大阪府たこ焼き市お好み焼き区33-4",
-    tel: "080-xxxx-xxxx",
-  },
-]);
-
+// TODO: リロードしなくても反映されるように
 const updateStatus = async (index: number) => {
-  if (!eventOrders.value[index].status) {
-    eventOrders.value[index].status = true;
+  const statusDocRef = doc(
+    db,
+    `users/${userId}/events/${eventId}/orders`,
+    handlingOrdersAndCustomers[index].order.docId
+  );
```

Comment:
```suggestion
  const statusDocRef = doc(
    db,
    `users/${userId}/events/${eventId}/orders`,
    handlingOrdersAndCustomers[index].order.docId
  ).withConverter(Firestore.converter(Firestore.Order));
```

伝え忘れていたのですが、このようにすることで型安全になります。
ためしにこれがない状態で `updateDoc` の `status` を数字とかに変えてみると、エラーがでないかなと想います。

Reply:
ありがとうございます。
そのように変更しました。